bindings/core/src/method/account.rs

```diff
@@ -32,27 +32,18 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 pub enum AccountMethod {
-    /// Burn native tokens. This doesn't require the foundry output which minted them, but will not increase
+    /// Burn outputs.
+
+    /// When burn **native tokens**. This doesn't require the foundry output which minted them, but will not increase
```

Comment:
Is something missing in the first sentence?

Reply:
"ing" and ":"? :)